proto/regen/ecocredit/marketplace/v1/types.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package regen.ecocredit.marketplace.v1;
+
+import "regen/ecocredit/marketplace/v1/state.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/marketplace";
+
+// AllowedDenomProposal is a gov Content type for approving a denom for use in
+// the marketplace.
+message AllowedDenomProposal {
+  option (gogoproto.goproto_stringer) = false;
+
+  // title is the title of the proposal.
+  string title = 1;
+
+  // description is the description of the proposal.
+  string description = 2;
+
+  // allowed_denom contains coin denom information that will be added to the
+  // allowed ask denoms list for use in the marketplace.
```

Comment:
It looks like we moved away from calling it an "ask denom" in preparation for creating buy orders that will have require the "bid denom" to be allowed but if we go this route lets keep it generic in the description.

```suggestion
  // allowed_denom contains coin denom information that will be added to the
  // list of allowed denoms for use in the marketplace.
```

Reply:
It looks like we say "ask/bid denom" in `AllowedDenom`, so maybe this should be more generic as well or both should be specific to ask denom and we can update to include bid denom when creating buy orders is available.